src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1303,14 +1306,14 @@ private MembersAndInitializers GetMembersAndInitializers()
             AddDeclarationDiagnostics(diagnostics);
             diagnostics.Free();
 
-            return membersAndInitializers;
+            return _lazyMembersAndInitializers!;
```

Comment:
>_lazyMembersAndInitializers [](start = 19, length = 27)

Was this change needed?

Reply:
Probably not. I think I misread the early return there.